src/Microsoft.AspNet.Mvc.ModelBinding/Binders/MutableObjectModelBinder.cs

```diff
@@ -52,7 +51,13 @@ protected virtual bool CanUpdateProperty(ModelMetadata propertyMetadata)
         internal async Task<bool> CanCreateModel(MutableObjectBinderContext context)
         {
             var bindingContext = context.ModelBindingContext;
+
+            // Reviewers: The isTopLevelObject value is wrong whenever a binder (e.g. CollectionModelBinder) uses
+            // GetMetadataForType(). Suggest we need ModelMetadata.GetMetadataForElementType() and a special case here
+            // for ComplexModelDto. Impact will likely be mainly extra "empty" (no bound property) objects created.
+            // Worth filing an issue?
```

Comment:
cool, Typically i have seen people commenting on PR itself to bring out something. I know typically != required :smiling_imp: